cibyl/sources/zuul/apis/utils/builds/__init__.py

```diff
@@ -16,6 +16,10 @@
 from enum import Enum
 
 
+class ArtifactError(Exception):
```

Comment:
This should inherit from SourceException instead of Exception.

Reply:
Ok, I will update it.

Reply:
Also forgot to document it.

Reply:
Fixed.